### PR TITLE
Done with my form

### PR DESCRIPTION
**Pull request by samoasao**
Hey team. The form is done. Please approve

### CODE REVIEW

**Review by stamm2911 — APPROVED** [review]

**Comment by samoasao** [comment]
Looks good

**Comment by stamm2911** [comment]
Looks good

File: index.html [hunk 1]
```diff
@@ -7,6 +7,11 @@
     <title>Document</title>
 </head>
 <body>
+
+    <form>
```
Comment (samoasao) [inline index.html:11]: This isn't good, it was supposed to be a checkbox

**Review by samoasao — COMMENTED** [review]
Eduardo please review again

**Pull request merged**